lua/lspconfig/jdtls.lua

```diff
@@ -110,6 +110,8 @@ configs[server_name] = {
           -- TODO: (steelsojka) Handle more than one edit?
           if action.command == 'java.apply.workspaceEdit' then
             action.edit = action.arguments[1]
+          elseif type(action.command) == 'table' and action.command.command == 'java.apply.workspaceEdit' then
+            action.edit = action.command.arguments[1]
```

Comment:
@jerrywang1981 
Why are there two cases of "action.command" and "action.command.command"?

Reply:
My guess is that it is because the response can either be an array of commands or an array of code actions:

```
    -- actions is (Command | CodeAction)[] | null
    -- CodeAction
    --      title: String
    --      kind?: CodeActionKind
    --      diagnostics?: Diagnostic[]
    --      isPreferred?: boolean
    --      edit?: WorkspaceEdit
    --      command?: Command
    --
    -- Command
    --      title: String
    --      command: String
    --      arguments?: any[]
```

If it is a codeaction, then the command text is within command.command.